views/components/HeroFooter/HeroFooter.css

```diff
@@ -0,0 +1,13 @@
+.footer_container {
+  width: 100%;
+  height: 88px;
+  align-self: flex-end;
+  display: flex;
+  align-items: center;
+}
+.footer_actions {
+  width: 1018px;
```

Comment:
we shouldn't need this.

Reply:
[9d7c4dc](https://github.com/AMP-SCZ/dpdash/pull/468/commits/9d7c4dc05024fa07956aaa578a84f844f590db07)